pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go

```diff
@@ -580,7 +579,7 @@ var _ = Describe("Validating VM Admitter", func() {
 								Bus: "scsi",
 							},
 						},
-						DedicatedIOThread: k8sptr.BoolPtr(true),
+						DedicatedIOThread: pointer.P(true),
```

Comment:
I know it is not your fault, but it would be nice if you could also remove the already present duplicate import:
```
	"kubevirt.io/kubevirt/pkg/pointer"
	virtpointer "kubevirt.io/kubevirt/pkg/pointer"
```
:)

Reply:
@fossedihelm I think I have fixed all of all duplication.

Reply:
@fossedihelm Done.